doc/effects.md

```diff
@@ -46,6 +46,24 @@ square.addEffect(ScaleEffect(
 ));
 ```
 
+## RotationalEffect
+
+Applied to `PositionComponent`s, this effect can be used to rotate the component, using an [animation curve](https://api.flutter.dev/flutter/animation/Curves-class.html).
+
+Usage example:
+```dart
+import 'dart:math';
+
+import 'package:flame/effects/effects.dart';
+
+// Square is a PositionComponent
+square.addEffect(RotationalEffect(
+  rotation: 2 * pi, // In radians
+  speed: 1.0,
```

Comment:
this is rad / s  recond? would be nice to add the unit in the example as well

Reply:
Fixed!